roles/app/defaults/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
+s3_headers: "{\"Cache-Control\":\"max-age=31557600\"}"
+s3_protocol: "https"
+
+attachment_url: "/spree/products/:id/:style/:basename.:extension"
+attachment_styles: "{\"mini\":[\"48x48>\",\"\"],\"small\":[\"100x100>\",\"\"],\"product\":[\"240x240>\",\"\"],\"large\":[\"600x600>\",\"\"]}"
```

Comment:
The small `100x100#` setting is actually a bug. It's the Spree default but OFN's design expects `112x112#` for the product listing. We are probably going to change this for the mobile design though.

Reply:
:+1:  agree, here I am just copying what's on the DB to the config file. Fixing this is out of scope for this PR. I created an issue for that: https://github.com/openfoodfoundation/openfoodnetwork/issues/5911